store/ssmstore.go

```diff
@@ -267,6 +267,7 @@ func (s *SSMStore) List(service string, includeValues bool) ([]Secret, error) {
 						Values: []*string{aws.String("/" + service)},
 					},
 				},
+				MaxResults: aws.Int64(50),
```

Comment:
Can you also add this `MaxResults` parameter to the `DescribeParametersInput` below for non-path based keys.

Reply:
You are right, I forgot about the other call. Updated